arkane/encorr/reference.py

```diff
@@ -665,5 +669,57 @@ def _is_valid_calculated_data(data_dictionary):
     return False
 
 
+def update_reference_db(path: str, model_chemistry: str = None, ref_paths: List[str] = None) -> None:
+    """
+    Update the reference species database from a collection of ArkaneSpecies YAML files. All of the YAML files should
+    be included in a single folder.
+
+    Notes:
+        ArkaneSpecies YAML files must end in either '.yml' or '.yaml'
+
+    Args:
+        path (str): The path to a directory containing all of the ArkaneSpecies YAML files
+        model_chemistry (str): Model chemistry string matching the Arkane format to be updated. If not supplied will
+            default to the ``level_of_theory`` parameter of the ArkaneSpecies objects
+        ref_paths (list): A list to the paths of reference sets to load. By default only the main reference set is
+            loaded
+
+    Returns:
+        None
+    """
+    spcs_list = []
+    for f in os.listdir(path):
+        if any(file_extension in f.lower() for file_extension in ['.yml', '.yaml']):
```

Comment:
This is super nitpicky and very much a personal preference, but it would save one line. I typically just loop through the output of calling `glob.glob` on `'*.yml'` and `'*.yaml'`. I guess that means you'd also have to import `glob`, so you wouldn't actually save a line... Feel free to ignore this comment completely.

Reply:
I'll add it, just because it is worth learning about glob so that I can use it again in the future when needed. Thanks for the suggestion!